### PR TITLE
buffs cyborg cleaner bottles and code cleanliness

### PR DESCRIPTION
**Pull request by ZWRh3**
## About the PR 
changes detectives luminol spray bottle examine text to be better

makes civilian borg module spray bottle regenerate 5u per cycle instead of 1u, buff its max capacity from 25u to 100u

patches a tiny bug of the cyborg bottle not refilling until you try to spray if you empty it via non attack means like smoke powder

tiny code improvements
## Why's this needed? 
makes the cleaning bottle cyborg item not painful to use
cleanliness
## Changelog

```changelog
(u)ZWRh3
(+) Civilian cyborg cleaning spray bottles refill speed has been increased from 1u to 5u, maximum volume changed from 25 to 100.
```

### CODE REVIEW

File: code/obj/item/mops_cleaners.dm [hunk 1]
```diff
@@ -32,49 +33,42 @@ WET FLOOR SIGN
...
 	..()
-	create_reagents(100)
+	create_reagents(initial_volume)
 
 /obj/item/spraybottle/detective
 	name = "luminol bottle"
 	desc = "A spray bottle labeled 'Luminol - Blood Detection Agent'. That's what those fancy detectives use to see blood!"
-
+	rc_flags = RC_VISIBLE | RC_SPECTRO | RC_SCALE
 	New()
```
Comment (frawhst) [inline code/obj/item/mops_cleaners.dm:42]:
```suggestion

	New()
```
slightly prettier

File: code/obj/item/mops_cleaners.dm [hunk 2]
```diff
@@ -32,49 +33,42 @@ WET FLOOR SIGN
...
 
-/obj/item/spraybottle/cleaner/robot/New()
-	..()
-	create_reagents(25)// no more 100 units on spawn for you, mister!
 /obj/item/spraybottle/cleaner/robot
 	name = "cybernetic cleaner spray bottle"
 	desc = "A cleaner spray bottle jury-rigged to synthesize space cleaner."
 	icon_state = "cleaner_robot"
-
+	var/refill_speed = 5
```
Comment (frawhst) [inline code/obj/item/mops_cleaners.dm:58]:
```suggestion
	var/refill_speed = 5
	
```
.

File: code/obj/item/mops_cleaners.dm [hunk 3]
```diff
@@ -32,49 +33,42 @@ WET FLOOR SIGN
...
 	disposing()
 		..()
 		processing_items.Remove(src)
 
-	afterattack(atom/A, mob/user)
-		. = ..()
-		if (src.reagents.total_volume < 25)
+	on_reagent_change()
+		..()
+		if (src.reagents.total_volume < reagents.maximum_volume)
```
Comment (frawhst) [inline code/obj/item/mops_cleaners.dm:65]:
```suggestion
		if (src.reagents.total_volume < src.reagents.maximum_volume)
```
consistency

File: code/obj/item/mops_cleaners.dm [hunk 4]
```diff
@@ -32,49 +33,42 @@ WET FLOOR SIGN
...
 			processing_items |= src
 
 	process()
 		..()
-		if (src.reagents.total_volume < 25)
-			src.reagents.add_reagent("cleaner", 1)
+		if (src.reagents.total_volume < reagents.maximum_volume)
```
Comment (frawhst) [inline code/obj/item/mops_cleaners.dm:70]:
```suggestion
		if (src.reagents.total_volume < src.reagents.maximum_volume)
```

**Review by frawhst — CHANGES_REQUESTED** [review]
Looks mostly good, but going from 1u regen rate to 5u seems a bit strong, and I'd rather not make civ borgs into the janitor version of mediborgs (ie way better than human janitors). 

I'd be fine with 2.5u regen rate and 50u max capacity, and then we can just buff it again if it's still annoying to use.

Also some minor style stuff.

**Comment by ZWRh3** [comment]
> Looks mostly good, but going from 1u regen rate to 5u seems a bit strong, and I'd rather not make civ borgs into the janitor version of mediborgs (ie way better than human janitors).
> 
> I'd be fine with 2.5u regen rate and 50u max capacity, and then we can just buff it again if it's still annoying to use.

done

**Review by frawhst — APPROVED** [review]

**Pull request merged**